README.md

```diff
@@ -72,7 +72,7 @@ import { myApiFetcher } from '.';
 it('should call the API', async () => {
   // injectable() needs the original implementation as first argument
   // and the replacement implementation as second
-  const fetchApiDi = injectable(fetchApi, jest.fn().mockResolvedValue('mock'));
+  const fetchApiDi = injectable(fetchApi, jest.fn().mockResolvedValue({ data: 'mock' }));
```

Comment:
not sure about typesafety of this command
Way more usual this is written as
```suggestion
  const fetchApiDi = injectable(fetchApi, () => ({ data: 'mock' }));
```
In such case magnetic can enforce correct types

Reply:
I think the downside of using `() => ({ data: 'mock' })` is that we cannot write `expect(fetchApiDi).toHaveBeenCalled()`. Here I wanted to showcase that the return of the injectable is the mock, so you can run expectations on it. 
But yeah, types will probably end up a bit loose